doc/getting-started/getting-started.md

```diff
@@ -0,0 +1,804 @@
+  await ledger.sign_and_submit_request(pool_handle, steward_wallet, steward_did, nym_request)
+```
+At this point **Faber** has DID related to his identity in the Ledger. 
+
+**Acme**, **Thrift Bank**, and **Government** must pass the same Onboarding process connection establishment with **Steward**.
+
+#### Claim Schemas Setup
+**Claim Schema** - is the base semantic structure that describes the list of attributes which one particular Claim can contain. 
+
+Note: It's not possible to update existing Schema. So, if the Schema needs to be evolved, a new Schema with a new version or name needs to be created.
+
+**Claim Schema** can be created and saved in the Ledger by any **Trust Anchor** by following the next steps:
+1. **Trust Anchor** optionally creates new DID record in his wallet and sends corresponded NYM transaction to the Ledger.
+1. **Trust Anchor** prepares **Claim Schema**.
+1. **Trust Anchor** sends corresponded Schema transaction to the Ledger by calling consistently ``ledger.build_schema_request`` to build Schema request and ``ledger.sign_and_submit_request`` to send created request.
+
+Here is **Government** creates and publishes **Transcript** Claim Schema to the Ledger:
+```python
+  (government_issuer_did, government_issuer_key) = await did.create_and_store_my_did(government_wallet, "{}")
+  nym_request = await ledger.build_nym_request(government_did, government_issuer_did, government_issuer_key, None, None)
+  await ledger.sign_and_submit_request(pool_handle, government_handle, government_did, nym_request)
+        
+  transcript_schema = {
+      'name': 'Transcript',
+      'version': '1.2',
+      'attr_names': ['first_name', 'last_name', 'degree', 'status', 'year', 'average', 'ssn']
+  }
+  schema_request = await ledger.build_schema_request(government_issuer_did, json.dumps(transcript_schema))
+  await ledger.sign_and_submit_request(pool_handle, government_wallet, government_issuer_did, schema_request)
+```
+
+The same way **Government** creates and publishes **Job-Certificate** Claim Schema to the Ledger:
+```python    
+  job_certificate_schema = {
+      'name': 'Job-Certificate',
+      'version': '0.2',
+      'attr_names': ['first_name', 'last_name', 'salary', 'employee_status', 'experience']
+  }
+  schema_request = await ledger.build_schema_request(government_issuer_did, json.dumps(to the Ledger))
+  await ledger.sign_and_submit_request(pool_handle, government_wallet, government_issuer_did, schema_request)
+```
+
+At this point we have **Transcript** and **Job-Certificate** Claim Schemas published by **Government** to the Ledger.
+
+#### Claim Definition Setup
+Claim Definition is similar to keys that Issuer use for signing of Claims satisfied specific Claim Schema. 
+
+Note: One Issuer DID can create just one Claim Definition for the specific Claim Schema, but different Issuer DIDs can create numerous Claim Definitions for the same Claim Schema.
```

Comment:
I suggest to remove this info as in current design of Revocation we are intended to break these rules.